documentation/Maintenance.md

```diff
@@ -495,7 +495,7 @@ For more information, see [Validation by JSON Schemas](Installation.md#validatio
 
 By default, no parameters are required. However, if necessary, you can specify custom.
 
-#### backup_location Parameter
+#### Backup_location Parameter
```

Comment:
Hi @shmo1218, thank you for fixing of the documentation.
Here backup_location as well as etcd, graceful_reboot, nginx-ingress-controller, migrate_cri, cri, yum-repositories, packages-associations are case sensitive keywords. I think they should be left as-is.
Also I would suggest to revert some changes from https://github.com/Netcracker/KubeMarine/pull/169/commits/e84ff71da48e398b6f0b3c4aa110f51d4330957e (for example, changes from "nodes Parameter" to "Nodes Parameter")
This approach is aligned with https://peps.python.org/pep-0008/#comments
> (never alter the case of identifiers!).

Reply:
Hi, @ilia1243,

Thank you for the feedback. I have rolled back the changes I had made to the identifier names in headings. Please take a look.

Thanks!